CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
 # CHANGELOG
 
 ## Not released
+
 - Pass actions and options to WrapperWidgetUI from HistogramWidget and CategoryWidget [#50](https://github.com/CartoDB/carto-react-lib/pull/50)
-- Fix addSource in cartoSlice spreading layerAttributes [#52∫](https://github.com/CartoDB/carto-react-lib/pull/52)
+- Fix addSource in cartoSlice spreading layerAttributes [#52](https://github.com/CartoDB/carto-react-lib/pull/52)
+- extend widget props to WrapperWifgetUI with a better implementation than in [#50](https://github.com/CartoDB/carto-react-lib/pull/50) [#57](https://github.com/CartoDB/carto-react-lib/pull/57)
```

Comment:
```suggestion
- Extend widget props to WrapperWidgetUI with a better implementation [#57](https://github.com/CartoDB/carto-react-lib/pull/57)
```

🔝 a typo in Widget and a suggestion to be shorter

Reply:
Let's go 🚀